### PR TITLE
Feature/get yesterdays winner

### PR DESCRIPTION
**Pull request by B-Griffinn**
# Get Yesterday's Winner

Yesterday's winner should be sent to the Front End up until today's voting ends and a new winner is announced.

### CODE REVIEW

**Comment by B-Griffinn** [comment]
I truly did not meant to format those two extra files

File: api/ranking/rankingRouter.js [hunk 1]
```diff
@@ -97,8 +98,14 @@ router.get("/histogram", restricted, async (req, res) => {
...
     let Today = await getPrompt();
-    if (Today.voting || Today.topThree || Today.active || !Today)
-      return res.status(400).json({ error: "No winners declared yet" });
+    let yesterdaysWinner;
+    if (Today.voting || Today.topThree || Today.active || !Today) {
+      // OLD CODE >> return res.status(400).json({ error: "No winners declared yet" });
+
+      // return the result of our getYesterdayWinner function
+      yesterdaysWinner = getYesterdaysWinner();
+      return res.status(200).json(yesterdaysWinner);
```
Comment (lgv-0) [inline api/ranking/rankingRouter.js]: Might be beneficial to add a `tomorrow: 1` flag of sorts so the FE has a signal to display that this is yesterdays winner they're seeing

File: api/ranking/rankingModel.js [hunk 2]
```diff
@@ -123,3 +124,15 @@ async function getWinner(winnerId) {
...
+/**
+ * SELECT *
+FROM submissions
+INNER JOIN winning_stories ON submissions.id = winning_stories.story_id
+ */
+// return the last item in our db("winning_stories")
+async function getYesterdaysWinner() {
+  return await db("winning_stories")
+    .join("winning_stories", "winning_stories.story_id", "submissions.id")
+    .first();
```
Comment (lgv-0) [inline api/ranking/rankingModel.js:141]: Says first but we want the last entry

**Review by lgv-0 — CHANGES_REQUESTED** [review]

**Comment by B-Griffinn** [comment]
@lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0 @lgv-0

**Review by lgv-0 — APPROVED** [review]

**Pull request merged**